packages/networks/src/defaults/ledger.ts

```diff
@@ -20,5 +20,6 @@ export const knownLedger: KnownLedger = {
   polymesh: 0x00000253,
   sora: 0x00000269,
   statemine: 0x000001b2, // common-good on Kusama, shares derivation
+  statemint: 0x00000162, // common-good on Polkadot, shares derivation
```

Comment:
As per the comment above that explains the additions here, additional information is required -

- The genesis has for the network should be added to `genesis.ts`
- The Ledger app should be added to `hw-ledger/defaults.ts`

Will approve the CI run (since it is a first contribution), but since it does not have the above, it will fail the checks.

Reply:
Yes, I saw that later. I'm fixing it. Thanks for the fast answer!